### DISCUSSION TITLE
What does the anchor button do in a text box of the annotation editor?

### DISCUSSION THREAD

**Discussion by XMB-7** [gildas-lormeau/SingleFile#1166] [Q&A] [answered]
I just found that the anchor icon is clickable and tried adding two text boxes and clicking on one to make it lighter than the other, but it doesn’t seem to make any difference.

![image](https://github.com/gildas-lormeau/SingleFile/assets/65625632/03e9b44a-c146-4672-b725-87b2e77b1c75)

**Comment by gildas-lormeau** [comment, marked as answer]
When enabled, the note can be attached to a node in the document. When disabled, the position of the note is fixed.
